specification/hugr.md

```diff
@@ -1612,12 +1632,9 @@ either differs from or is not part of the
 [WebAssembly](https://webassembly.github.io/spec/core/exec/numerics.html)
 specification.
 
-Const nodes:
+### `arithmetic.int`
```

Comment:
```suggestion
#### `arithmetic.int`
```

Reply:
Rather btw, but this suggests to me that it should be `arithmetic.int` and `arithmetic.int.ops` rather than `...types` and `...`. Never mind...